.travis.yml

```diff
@@ -1,3 +1,5 @@
-language: node_js
-node_js:
-  - "10.9"
+language: ruby
```

Comment:
Any reason you went for Travis? Absolutely your choice, but I think on the whole we default to CircleCI at this point.

Reply:
No reason. Didn't even think about it. Will move it over.